common/reviews/api/operation-graph.api.md

```diff
@@ -33,7 +33,7 @@ export interface IExecuteOperationContext extends Omit<IOperationRunnerContext,
     afterExecute(operation: Operation, state: IOperationState): void;
     beforeExecute(operation: Operation, state: IOperationState): void;
     queueWork(workFn: () => Promise<OperationStatus>, priority: number): Promise<OperationStatus>;
-    requestRun?: (requestor?: string) => void;
+    requestRun?: (requester: string) => void;
```

Comment:
This is a breaking API change. This should at least be a minor bump and these should be called out in the changelog, with a "(BREAKING API CHANGE) " prefix.